doc/uftrace-record.md

```diff
@@ -359,6 +362,25 @@ example will show all user functions and the (kernel) page fault handler.
       3.340 us [14721] |   } /* a */
      79.086 us [14721] | } /* main */
 
+In addition, you can set filter to record selected source locations with `-L` option.
+
+  $ uftrace record -L s-libmain.c t-lib
+  $ uftrace replay --srcline
+  # DURATION     TID     FUNCTION [SOURCE]
+              [  5043] | main() { /* /home/uftrace/tests/s-libmain.c:16 */
+     6.998 us [  5043] |   foo(); /* /home/uftrace/tests/s-libmain.c:11 */
+     9.393 us [  5043] | } /* main at /home/uftrace/tests/s-libmain.c:16 */
```

Comment:
```
/* main at /home/uftrace/tests/s-libmain.c:16 */
```
This should be changed to
```
/* main */
```

Reply:
Right, the return path used to have a different location info than the beginning of the function and it's possible to have multiple return locations.  As we don't know exactly where it returns from, we can just print function name here.